### ISSUE TITLE
[BUG] dfreds integration is no longer working.

### ISSUE THREAD

**Issue by kwilli96** [ironmonk88/monks-active-tiles#831]
**Describe the bug**
I tried to just create a fix in github and push it to you but there is an issue with the merge. I think it had to do with new lines but it doesn't matter. Essentially you were getting dfreds effects using game.dfreds.effects.all and game.dfreds.effectInterface.findEffectByName but they seem to no longer exist or at least don't exist in my system. 

Here are the changes that I used:

Line: 8943
```
const foundEffect = game.dfreds.effectInterface.findEffect({effectName:action.data.effect});
```

Lines: 8983-8987
```
list: () => {
      return game.dfreds.effectInterface.findEffects().map((effect) => {
            return i18n(effect.name);
       }).sort((a, b) => a.localeCompare(b));
},
```

Lines: 8917-8921
```
list: () => {
      return game.dfreds.effectInterface.findEffects().map((effect) => {
            return i18n(effect.name);
       }).sort((a, b) => a.localeCompare(b));
},
```

**To Reproduce**
1. Set an active tile action to convenient effects
2. Nothing shows up.

**Please complete as much of the following information as possible:**
 - Browser [chrome]
 - Foundry Version: v12.331
 - System: D&D5e v3.3.1
 - Module version: v12.02
 - Dfreds Convenient Effects version: 
 - Are there any errors in the console: Yes
1. "all" on undefined effects: 8983-8987 and  8917-8921
2. getEffectByName is not a defined function line: 8943

**Comment by phre3d** [comment]
Not sure if its the same issue but i just tried it with DFCE and there's no box to select the specific effect (see attached)
![DF-CE 1](https://github.com/user-attachments/assets/a4b55673-c560-4754-aba6-e56b8746e9c3)

**Comment by kwilli96** [comment]
> Not sure if its the same issue but i just tried it with DFCE and there's no box to select the specific effect (see attached)

@phre3d Yeah that is the issue that I found. If you are wanting to fix it you can make the changes to this file: %LOCALAPPDATA%\FoundryVTT\Data\modules\monks-active-tiles\actions.js

**Comment by phre3d** [comment]
Thanks, I'll try it tomorrow!
